release-notes/aws/v11.0.2.md

```diff
@@ -0,0 +1,28 @@
+## :zap: Giant Swarm Release 11.0.2 for AWS is now active for you! :zap:
+
+This release includes some reliability and user experience improvements.
+
+Below, you can find more details on components that were changed with this release.
+
+### cert-manager v0.9.0 ([Giant Swarm app v1.0.6](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#v106-2020-02-28))
+
+- Configured app icon.
+
+### chart-operator [v0.12.0](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#v0120-2020-03-09)
+
+- Added chartmigration resource for migrating from chartconfig to chart CRs.
+- Updated release resource.
+  - Do not wait when installing or updating long running Helm releases.
+  - Use version field from chart CR to reduce number of HTTP requests to pull chart tarballs.
+  - Wait for deleted Helm release before removing finalizer. 
+- Updated status resource.
+  - Improve reason field in CR status when installing a chart fails.
+- Removed legacy chartconfig controller.
+
+### cluster-operator [v0.23.2](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.2)
```

Comment:
I'm confused I saw cluster-operator 0.23.2 had been deleted from aws-app-collection. https://github.com/giantswarm/aws-app-collection/tree/master/helm/aws-app-collection-chart/templates

Reply:
We were running too many cluster-operator pod. So we didn't have enough capacity in axolotl and some other CPs.

So I checked the cluster-operator versions not in use and deleted them. This is manual cleanup needed until release-operator creates the app CRs on demand.

I think we should use cluster-operator 0.23.4 here.

@sslavic Can you confirm since you created 0.23.4.